otterize-kubernetes/values.yaml

```diff
@@ -5,6 +5,7 @@ deployment:
   networkMapper: true
 
 global:
+  useOtterizeCloudCredentials: false
```

Comment:
'global.useOtterizeCloudCredentials' is relevant only in credentials-operator (and spire-deployment), but 'global.otterizeCloud.credentials' is used by credentials-operator and intents-operator. It makes 'global.useOtterizeCloudCredentials' a little bit misleading because when set  to false, intents-operator will still use 'global.otterizeCloud.credentials'